### PR TITLE
Completed html-forms.

### PR DESCRIPTION
**Pull request by alexdenbaugh**
html-forms-empty
html-forms-full
html-forms-submit

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="" method="GET">
+    <label for="email">Email</label>
+    <div id="email">
```
Comment (codingcodymiller) [inline html-forms/index.html:13]: This `id` should be on the `input` to match with the `label`

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      <input type="text" name="email">
+    </div>
+    <label for="why-coding">Why is coding cool?</label>
+    <div id="why-coding">
```
Comment (codingcodymiller) [inline html-forms/index.html:17]: This `id` should be on the `select` to match with the `label`

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        <option value="coding-interesting">Because it's interesting</option>
+        <option value="coding-fun">Because it's fun</option>
+        <option value="coding-powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <label for="favorite-language">Favorite Language (choose one)</label>
+    <div id="favorite-language">
+      <input type="radio" name="favorite-language" value="html"> HTML
+      <input type="radio" name="favorite-language" value="css"> CSS
+      <input type="radio" name="favorite-language" value="javascript"> JavaScript
```
Comment (codingcodymiller) [inline html-forms/index.html:29]: The text for each option should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <label for="sign-up">Would you like to sign up for more information?</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:31]: This doesn't need to be a `label`, it doesn't match to one specific form control element.

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+    <label for="why-coding">Why is coding cool?</label>
+    <div id="why-coding">
+      <select name="why-coding">
+        <option value="select-option">Select an option</option>
+        <option value="coding-interesting">Because it's interesting</option>
+        <option value="coding-fun">Because it's fun</option>
+        <option value="coding-powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <label for="favorite-language">Favorite Language (choose one)</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:25]: This doesn't need to be a `label`, it doesn't match to one specific form control element.

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+    <div id="favorite-language">
+      <input type="radio" name="favorite-language" value="html"> HTML
+      <input type="radio" name="favorite-language" value="css"> CSS
+      <input type="radio" name="favorite-language" value="javascript"> JavaScript
+    </div>
+    <label for="sign-up">Would you like to sign up for more information?</label>
+    <div id="sign-up">
+      <input type="checkbox" name="sign-up" value="sign-up-yes"> Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:33]: The text labelling the checkbox should be in a `label` element. Make sure you use a `for` attribute with a value that matches the `id` of the form control element this is labelling!

File: html-forms/index.html [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+    </div>
+    <label for="comments">Comments</label>
+    <div id="comments">
```
Comment (codingcodymiller) [inline html-forms/index.html:36]: This `id` should be on the `textarea` to match with the `label`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work so far!! Please see my notes

File: html-forms/index.html (hunk 1 above)
Comment (alexdenbaugh) [reply html-forms/index.html:13]: Okay what about for the radio inputs where there are multiple options?  Do we just put the id on the first input?

**Review by codingcodymiller — APPROVED** [review]
Nice work!!

**Pull request merged**